CODE.md

```diff
@@ -0,0 +1,114 @@
+Explain how we have our own code standards configured in Codacy based on the documentation below.
+
+We should talk about code test coverage that may also be tested by Codacy.
+
+I'd like to discuss how to use and install linters and configs for linters. Maybe I'll discuss that in each language section. :fire:
+
+## Languages :sparkles:
+Here are the code standards for different languages. We also have links to linters that are supported by Codacy. You can add these linters to your projects. :+1:
+
+### PHP :older_man:
+
+#### Code Style
+* [PEAR](https://pear.php.net/manual/en/standards.php)
+#### Linter
+* [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer)
+#### Test Coverage
+* [PHP Code Coverage for Codacy](https://github.com/codacy/python-codacy-coverage)
+
+### Javascript :godmode: :trophy:
+
+#### Code Style
+* [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript)
+#### Linters / Config
+* [ESLint](https://eslint.org/)
+* [GCTools ESLint Config Package](https://github.com/gctools-outilsgc/gctools-components/tree/master/packages/eslint-config)
+#### Test Coverage
+* [Code Coverage for Codacy](https://github.com/codacy/node-codacy-coverage)
+
+### Python :snake:
+
+#### Code Style
+* [PEP 8](https://www.python.org/dev/peps/pep-0008/)
+#### Linter
+* [Pylint](https://www.pylint.org/)
+#### Test Coverage
+* [Code Coverage for Codacy](https://github.com/codacy/python-codacy-coverage)
+
+### CSS / Sass :art:
+
+#### Code Style
+* [Airbnb Style Guide](https://github.com/airbnb/css)
+#### Linters
+* [CSSLint](https://github.com/CSSLint/csslint)
+* [Stylelint](https://stylelint.io/)
+
+### Go :rat: (no gopher emoji :cry:)
+
+#### Code Style
+* [Effective Go](https://golang.org/doc/effective_go.html)
+* [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments)
+#### Linter
+* [Golint Repo](https://github.com/golang/lint)
+#### Test Coverage
+* [Code Coverage for Codacy](https://github.com/schrej/godacov)
+
+### Dockerfile :whale:
+
+#### Style
+* [Dockerfile Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)
+#### Linter
+* [Hadolint](https://github.com/hadolint/hadolint)
+
+## Document Naming Convention :surfer:
+
+### Design Working Files (.ai .psd .pdf) / Images / Folders / Git Branches :page_facing_up:
+
+:clap: ALL LOWERCASE :clap:
+:clap: Underscores '_' instead of spaces or dashes :clap:
+
+#### Versioning :clipboard:
+Major versions 
+* v01
+* v02
+* v03
+* etc
+
+Minor changes to version
+* v01_1
+* v01_2
+* etc
+
+A working design document on its second major version with a minor change:
+blog_home_v02_1.ai
+
+#### Folders :open_file_folder:
+Folder names should be descriptive, maybe something about folder structure?
+working_files
+
+#### Assets :camera:
+happy_cat.jpg
+logo.png
+
+
+#### Git Branches :deciduous_tree:
+Be descriptive with the purpose of the branch name. Use underscores '_' for spaces

```

Comment:
When creating a branch through GitHub Desktop it will automatically replace spaces with dash `-`. Allows no effort or thought being required for the result. So I would prefer dash used for spacing, or both being acceptable. 
![dash](https://user-images.githubusercontent.com/34379222/42041959-07e20680-7ac1-11e8-8ff8-ddd2322c125e.PNG)

Reply:
Yeah, from my experience in github at least the standard is dashes.

Reply:
I'm good with using dashes. I would like us all to agree on one standard, I don't want us to interchange them, so I'm cool with using the github standard 🤙